episode-10-switching-aws-to-roar/app/representers/product_representer.rb

```diff
@@ -0,0 +1,30 @@
+require 'roar/decorator'
+# require 'roar/json/json_api'
+
+class ProductRepresenter < Roar::Decorator
+  include Roar::JSON::JSONAPI
+  # include Roar::Hypermedia
+
+  type :product
+
+  property :id
+  property :name
+  property :description
+  # property :product_type, as: :type
+  property :apv
+  property :image_url
+
+  # has_one :manufacturer
```

Comment:
f I enable this I get:

NoMethodError: undefined method _links' for #<Product:0x007f8fdeee00c8> from /Users/psturgeon/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activemodel-5.0.0.1/lib/active_model/attribute_methods.rb:433:inmethod_missing'